orcid-core/src/main/java/org/orcid/pojo/ajaxForm/AffiliationForm.java

```diff
@@ -30,8 +33,12 @@
 import org.orcid.jaxb.model.v3.dev1.record.AffiliationType;
 import org.orcid.jaxb.model.v3.dev1.record.Education;
 import org.orcid.jaxb.model.v3.dev1.record.Employment;
+import org.orcid.pojo.OrgDisambiguated;
 
 public class AffiliationForm implements ErrorsInterface, Serializable {
+    
+    @Resource
+    private OrgDisambiguatedManager orgDisambiguatedManager;
```

Comment:
Hey @lizkrznarich, you don't need the OrgDisambiguatedManager here, it looks you don't use it anywhere and, as long as it is possible, we want to keep form elements as clean as possible and with no references to the managers, managers should be called from controllers only.